test-poc/framework/src/main/java/org/keycloak/test/framework/injection/ValueTypeAlias.java

```diff
@@ -0,0 +1,23 @@
+package org.keycloak.test.framework.injection;
+
+import org.keycloak.test.framework.server.KeycloakTestServer;
+import org.openqa.selenium.WebDriver;
+
+import java.util.Map;
+
+public class ValueTypeAlias {
+
+    private static final Map<Class, String> aliases = Map.of(
+            WebDriver.class, "browser",
+            KeycloakTestServer.class, "server"
```

Comment:
@stianst Since TestDatabase is merged, I propose to rebase it and add a database support here as well with particular aliases. WDYT?